src/backends/mysql/statement.cpp

```diff
@@ -402,24 +402,47 @@ int mysql_statement_backend::prepare_for_describe()
 }
 
 void mysql_statement_backend::describe_column(int colNum,
-    data_type & type, std::string & columnName)
+    data_type & type, db_type & dbtype, std::string & columnName)
 {
     int pos = colNum - 1;
     MYSQL_FIELD *field = mysql_fetch_field_direct(result_, pos);
     switch (field->type)
     {
     case FIELD_TYPE_CHAR:       //MYSQL_TYPE_TINY:
+        type = dt_integer;
+        dbtype = field->flags & UNSIGNED_FLAG ? db_uint8 : db_int8;
```

Comment:
Should I do get<int> or get<int8_t> ?

Reply:
This is discussed in #1088 so let's keep it there to reduce confusion.